inc/form.class.php

```diff
@@ -821,7 +824,8 @@ public static function getFormList(int $rootCategory = 0, string $keywords = '',
                'description'      => '',
                'type'             => 'faq',
                'usage_count'      => $faq['view'],
-               'is_default'       => false
+               'is_default'       => false,
+               'tile_template'    => 'fix'
```

Comment:
I think I missed this in the previuous review.

```suggestion
               'tile_template'    => PluginFormcreatorEntityConfig::CONFIG_UI_FORM_MASONRY
```